Package.swift

```diff
@@ -0,0 +1,33 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "BokenEngine",
+    platforms: [.iOS(.v13)],
```

Comment:
```
platforms: [
    `.macOS(.v10_14), .iOS(.v13)`
],
```

@yeraydavidrodriguez we are supporting macOS

Reply:
Done